src/common/utils/recommend.ts

```diff
@@ -92,9 +92,13 @@ export function getFedLevel(projection: Projection): FedLevel | null {
 
   if (weeklyCasesPer100k > 100 && positiveTestRate > 0.1) {
     return FedLevel.RED;
+  } else if (weeklyCasesPer100k > 100 || positiveTestRate > 0.1) {
+    return FedLevel.YELLOW;
   } else if (
-    (10 < weeklyCasesPer100k && weeklyCasesPer100k < 100) ||
-    (0.05 <= positiveTestRate && positiveTestRate <= 0.1)
+    10 < weeklyCasesPer100k &&
+    weeklyCasesPer100k < 100 &&
```

Comment:
should probably be `<=` else there's an edge case where if weeklyCasesPer100k is 100, you'll end up green.